projects/storefrontlib/src/lib/product/components/product-details/container/product-details.component.html

```diff
@@ -2,57 +2,13 @@
   <ng-container *cxOutlet="outlets.PAGE">
     <div class="container">
       <div class="cx-product-details">
-        <div>
-          <ng-container *cxOutlet="outlets.TITLE">
-            <div class="cx-product-details__item-name">{{ product?.name }}</div>
-            <div class="cx-product-details__item-code">
-              ID {{ product?.code }}
-            </div>
-          </ng-container>
-        </div>
-        <div class="cx-product-details__item-images">
-          <ng-container *cxOutlet="outlets.IMAGES">
-            <cx-product-images [product]="product"></cx-product-images>
-          </ng-container>
-        </div>
-        <div>
-          <ng-container *cxOutlet="outlets.RATING">
-            <div class="cx-product-details__item-rating">
-              <cx-star-rating
-                [rating]="product?.averageRating"
-                [disabled]="true"
-              ></cx-star-rating>
-              <div class="cx-product-details__review-count">
-                ({{ product?.numberOfReviews }})
-              </div>
-              <a
-                class="cx-product-details__review-link btn-link"
-                (click)="goToReviews($event)"
-              >
-                Write a review</a
-              >
-            </div>
-          </ng-container>
-
-          <ng-container *cxOutlet="outlets.SUMMARY">
-            <cx-product-summary
-              [product]="product"
-              (itemCountChange)="itemCount = $event"
-            ></cx-product-summary>
-          </ng-container>
-
-          <ng-container *cxOutlet="outlets.ADDTOCART">
-            <cx-add-to-cart
-              *ngIf="product?.stock?.stockLevelStatus !== 'outOfStock'"
-              [quantity]="itemCount"
-              [productCode]="product?.code"
-            ></cx-add-to-cart>
-          </ng-container>
-
-          <ng-container *cxOutlet="outlets.SHARE">
-            <a href="#" class="btn-link"> Share </a>
-          </ng-container>
-        </div>
+        <ng-container *cxOutlet="outlets.SUMMARY">
+          <cx-product-summary [product]="product">
+            <ng-container *cxOutlet="outlets.IMAGES"
```

Comment:
nothing wrong here, but I'd prefer to move the outlet container to `product-summary.component.html`